CONTRIBUTING.md

```diff
@@ -533,6 +541,10 @@ it should ouput something like
  [X] MigrationName1575633316367
 ```
 
+# Running Remotely
+
+When not running locally, the client needs to be passed the server's location.  This can be done by changing your _.env_ file to include `NEXT_PUBLIC_APOLLO_SERVER=<http://address.of.graphql.server>`.  For example, if you started **_Chapter_** with `npm run both` and hosted it on `http://demo.chapter.org` then the address will be `http://demo.chapter.org:5000`.
```

Comment:
@ojeytonwilliams this seems like a good start.

The only change I'd suggest is linking ".env file" to the .env section of the same file using the link to #env-configuration-file, like

[_.env_ file](#env-configuration-file)